packages/nextjs/utils/scaffold-eth/contract.ts

```diff
@@ -8,11 +8,20 @@ import {
 } from "abitype";
 import type { ExtractAbiFunctionNames } from "abitype";
 import { Address, Log, TransactionReceipt } from "viem";
-import { Prettify } from "viem/dist/types/types/utils";
 import { UseContractEventConfig, UseContractReadConfig, UseContractWriteConfig } from "wagmi";
 import contractsData from "~~/generated/deployedContracts";
 import scaffoldConfig from "~~/scaffold.config";
 
+/**
+ * @description Combines members of an intersection into a readable type.
+ * @example
+ * Prettify<{ a: string } & { b: string } & { c: number, d: bigint }>
+ * => { a: string, b: string, c: number, d: bigint }
+ */
+type Prettify<T> = {
+  [K in keyof T]: T[K];
+} & unknown;
```

Comment:
Using `unkown` here instead of `{}` since we have eslint rule setup : 

![Screenshot 2023-09-24 at 9 23 08 PM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/5dc86538-58b5-4e8c-bb94-d5196e2c607d)


Also  using `unknown` is completely fine too as  suggested here : 
![Screenshot 2023-09-24 at 9 47 23 PM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/ec27f942-5ff8-4fe9-ae12-30043957f428)

OG post => https://www.totaltypescript.com/concepts/the-prettify-helper